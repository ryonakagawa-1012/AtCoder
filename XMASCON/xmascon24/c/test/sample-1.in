3
4 2000 1000000
2
2 3
3 2 6 4 5 1
