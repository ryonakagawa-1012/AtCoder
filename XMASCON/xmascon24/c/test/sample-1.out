9006004 12004000 36012000 48008000 66010000 74002000
